File: 3 КУРС/2 семестр/Java/main/java/com/example/voiceassistent/ForecastApi.java
package com.example.voiceassistent;
import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface ForecastApi {
    @GET("/current?access_key=dca34aad5d153452c096a2380ad963e1")
    Call<Forecast> getCurrentWeather(@Query("query") String city);
}

File: 3 КУРС/2 семестр/Учебная практика/asd/JavaProects/app/src/main/java/com/example/voiceassistent/ForecastAPI.kt
package com.example.voiceassistent

import retrofit2.http.GET
import retrofit2.http.Query

interface ForecastAPI {
    @GET("/current?access_key=f853c8a905f351f18c211114ddac44f1")
    open fun getCurrentWeather(@Query("query") city: String?): retrofit2.Call<Forecast?>?
}

//f853c8a905f351f18c211114ddac44f1